docs/content/getting-started/configuration-overview.md

```diff
@@ -64,6 +64,12 @@ You can override this using the `configFile` argument.
 traefik --configFile=foo/bar/myconfigfile.yml
 ```
 
+!!! important "_static_ & _dynamic_ configuration cannot be in the same file"
+
+    In order to provide _dynamic configuration_ from a file, you'll need to create an other file,
+    reference it in your _static configuration_ file using the [File provider](../providers/file.md).
+    Any _dynamic configuration_ seen in the _static configuration_ file is **discarded without warnings**.
+
```

Comment:
This is not true, the dynamic part of the configuration is not discarded, it is just not interpreted since that, on startup, Traefik is only looking for static configuration, the parsing simply ignores what is not recognized as static configuration.

Reply:
I've  updated the PR taking into account your comment.